.github/workflows/build_and_push.yml

```
@@ -27,6 +27,9 @@ jobs:
           node-version: '16'
 
       - name: Run build
+        env:
+          PUBLIC_URL: 'https://next-dev.kbase.us/services/static'
```

should actually just be a path, without the domain.
```suggestion
          PUBLIC_URL: '/services/static'
```